deployment/ansible/group_vars/all

```diff
@@ -29,8 +29,8 @@ papertrail_log_files:
     - "/var/log/event-feed.log"
     - "/var/log/upstart/otp.log"
 
-postgresql_version: "9.6"
-postgresql_package_version: "9.6.*"
+postgresql_version: "12"
+postgresql_package_version: "12.*.*"
 postgresql_support_libpq_version: "*"
```

Comment:
We updated our production database to Postgresql 12, [see work here](https://github.com/azavea/cac-tripplanner/issues/1298#issuecomment-1014865329), but we didn't update our local version along with it. Django 4+ has [now dropped support for Postgresql 9.6](https://docs.djangoproject.com/en/5.0/releases/4.0/#dropped-support-for-postgresql-9-6), so we need to update locally to avoid breaking provisioning -- and good to sync with our production env version anyways.

Reply:
I couldn't get `12.*.*` to work correctly to provision the `database` VM. Changed this to:

```ruby
postgresql_package_version: "12.*.pgdg20.04+1"
```

which is inline with other projects, [e.g. MMW](https://github.com/WikiWatershed/model-my-watershed/blob/5e44175e612402b89bfb6e00b5fddc624555e11e/deployment/ansible/group_vars/all#L17-L18), and it worked.

Reply:
Addressed in [cc65889](https://github.com/azavea/cac-tripplanner/pull/1380/commits/cc65889642499dda07504b41c77af552057481ed)